mmdet/datasets/coco.py

```diff
@@ -119,6 +119,7 @@ def parse_data_info(self, raw_data_info: dict) -> Union[dict, List[dict]]:
         else:
             seg_map_path = None
         data_info['img_path'] = img_path
+        data_info['file_name'] = img_info['file_name']
```

Comment:
file_name should be unnecessary since img_path already indicated that.

Reply:
The `image path` can be an absolute path. We should find another way to fix this potential error